v2/stacks/common.mk

```
@@ -40,7 +40,7 @@ verify-service: $(LOCAL_ENV_FILE)
 			[[ "$(SERVICE)" = $$svc ]] && exit 0;					\
 		done;										\
 	done;											\
-	exit 4
+	echo "\033[31m""service $(SERVICE) does not exist in stack. Failed to verify.\033[0m" && exit 4;
```

```suggestion
	echo "\033[31m""service $(SERVICE) does not exist in stack. Failed to verify.\033[0m" >&2
	@exit 4
```

- error messages should go to stderr (`>&2`)
- exit should be clear (on its own line)